sdk/storage/azure-storage-blob/tests/perfstress_tests/T1_legacy_tests/upload_block.py

```diff
@@ -0,0 +1,26 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import uuid
+
+from azure_devtools.perfstress_tests import RandomStream, get_random_bytes
+
+from ._test_base_legacy import _LegacyContainerTest
+
+
+class LegacyUploadBlockTest(_LegacyContainerTest):
+    def __init__(self, arguments):
+        super().__init__(arguments)
+        self.blob_name = "blobblocktest-" + str(uuid.uuid4())
+
+    def run_sync(self):
+        self.service_client.put_block(
+            container_name=self.container_name,
+            blob_name=self.blob_name,
+            block=get_random_bytes(self.args.size),
+            block_id=str(uuid.uuid4()))
```

Comment:
Can we put to the same block id each time, or do they need to be unique each time?

Reply:
Turns out it can be the same. Updating :)